app/common/src/cljc/teet/util/date.cljc

```diff
@@ -25,13 +22,25 @@
      (java.util.Date.)))
 
 
-
 (defn date-after?
   "compare dates as timestamps (NOT calendar days)"
   [a b]
   #?(:clj (.after a b)
      :cljs (> a b)))
 
+#?(:clj
+   (defn inc-days [date days]
+     (-> date
+         .getTime
+         (+ (* days 24 60 60 1000))
+         Date.)))
+
+#?(:clj
+   (defn dec-days [date days]
+     (-> date
+         .getTime
+         (- (* days 24 60 60 1000))
+         Date.)))
```

Comment:
Since we're using `clj-time`, here's another possible implementation:

```
(defn inc-days [date days]
  (-> date
      tc/from-date
      (t/plus (t/days days))
      tc/to-date))
```
where `t = clj-time.core` and `tc = clj-time.coerce`. More of an FYI in case we need to do some heavier date juggling in the future.